includes/gateways/stripe/includes/admin/admin-actions.php

```diff
@@ -388,3 +458,102 @@ function give_stripe_show_currency_notice() {
 }
 
 add_action( 'admin_notices', 'give_stripe_show_currency_notice' );
+
+/**
+ * Disconnect Stripe Account.
+ *
+ * @since 2.7.0
+ *
+ * @return void
+ */
+function give_stripe_disconnect_connect_stripe_account() {
+
+	$get_data = give_clean( $_GET );
+
+	if ( is_admin() && isset( $get_data['stripe_disconnected'] ) ) {
+		$account_name = ! empty( $get_data['account_name'] ) ? $get_data['account_name'] : false;
+
+		// Disconnect Stripe Account.
+		give_stripe_disconnect_account( $account_name );
+	}
+}
+
+add_action( 'admin_init', 'give_stripe_disconnect_connect_stripe_account' );
+
+/**
+ * Set default Stripe account.
+ *
+ * @since 2.7.0
+ *
+ * @return void
+ */
+function give_stripe_set_account_default() {
+
+	$post_data    = give_clean( $_POST );
+	$account_slug = ! empty( $post_data['account_slug'] ) ? give_clean( $post_data['account_slug'] ) : false;
+
+	// Update default Stripe account.
+	$is_updated = give_update_option( '_give_stripe_default_account', $account_slug );
+
+	if ( $is_updated ) {
+		wp_send_json_success();
+	} else {
+		wp_send_json_error();
+	}
+
+	give_die();
+}
+
+add_action( 'wp_ajax_give_stripe_set_account_default', 'give_stripe_set_account_default' );
+
+/**
+ * This function is used to update account name.
+ *
+ * @since 2.7.0
+ *
+ * @return void
+ */
+function give_stripe_update_account_name() {
+
+	$post_data        = give_clean( $_POST );
+	$account_slug     = ! empty( $post_data['account_slug'] ) ? $post_data['account_slug'] : false;
+	$new_account_name = ! empty( $post_data['new_account_name'] ) ? $post_data['new_account_name'] : false;
+
+	if ( ! empty( $account_slug ) && ! empty( $new_account_name ) ) {
+		$accounts             = give_stripe_get_all_accounts();
+		$account_keys         = array_keys( $accounts );
+		$account_values       = array_values( $accounts );
+		$new_account_slug     = give_stripe_convert_title_to_slug( $new_account_name );
+		$default_account_slug = give_stripe_get_default_account_slug();
+
+		// Bailout, if Account Name already exists.
+		if ( in_array( $new_account_slug, $account_keys, true ) ) {
+			wp_send_json_error( [ 'message' => esc_html__( 'This account name is already in use. Please enter a different account name.', 'give' ) ] );
+			return;
+		}
+
+		$key                  = array_search( $account_slug, $account_keys, true );
+		$account_keys[ $key ] = $new_account_slug;
+
+		$new_accounts = array_combine( $account_keys, $account_values );
+
+		// Update accounts.
+		give_update_option( '_give_stripe_get_all_accounts', $new_accounts );
+
+		if ( $account_slug === $default_account_slug ) {
+			give_update_option( '_give_stripe_default_account', $new_account_slug );
+		}
+
+		$success_args = [
+			'message' => esc_html__( 'Account Name updated successfully.', 'give' ),
+			'name'    => $new_account_name,
+			'slug'    => $new_account_slug,
+		];
+		wp_send_json_success( $success_args );
+	} else {
```

Comment:
@mehul0810 No need of `give_die` and `clean` condition because wp_send_json_success calls wp_die internally.